Server/src/Server.Infrastructure/Persistence/Migrations/20230927144414_Init.cs

```diff
@@ -25,7 +25,7 @@ protected override void Up(MigrationBuilder migrationBuilder)
                 },
                 constraints: table =>
                 {
-                    table.PrimaryKey("PK_Addresses", x => x.Id);
```

Comment:
Wydaje mi się, że możemy dodawać takie coś do konfiguracji encji. Póki każdy po zmianie bazy robi nową jedyną migrację, to trzeba by za każdym razem ręcznie zmieniać te nazwy.